pay-api/src/pay_api/config.py

```diff
@@ -133,19 +130,22 @@ class _Config():  # pylint: disable=too-few-public-methods
     # Auth API Endpoint
     AUTH_API_ENDPOINT = _get_config('AUTH_API_ENDPOINT')
 
-    # Sentry Config
-    SENTRY_DSN = _get_config('SENTRY_DSN', default=None)
+    # REPORT API Settings
+    REPORT_API_BASE_URL = f'{_get_config("REPORT_API_URL")}/reports'
 
     # BCOL Service
     BCOL_API_ENDPOINT = _get_config('BCOL_API_ENDPOINT')
 
+    # Sentry Config
+    SENTRY_DSN = _get_config('SENTRY_DSN', default=None)
+
     # Valid Payment redirect URLs
     VALID_REDIRECT_URLS = [(val.strip() if val != '' else None)
                            for val in _get_config('VALID_REDIRECT_URLS', default='').split(',')]
 
     # Service account details
-    KEYCLOAK_SERVICE_ACCOUNT_ID = _get_config('KEYCLOAK_SERVICE_ACCOUNT_ID')
-    KEYCLOAK_SERVICE_ACCOUNT_SECRET = _get_config('KEYCLOAK_SERVICE_ACCOUNT_SECRET')
+    KEYCLOAK_SERVICE_ACCOUNT_ID = os.getenv('SBC_AUTH_ADMIN_CLIENT_ID')
```

Comment:
I hope its ok to change from _get_config to os.getenv

Reply:
Fixed

Reply:
fixed